### ISSUE TITLE
UHD image not going to /opt...

### ISSUE THREAD

**Issue by DeerStalker25** [lmesserStep/LibreSDRB210#1]
Why does when running sudo /usr/lib/uhd/utils/uhd_images_downloader.py  dump into /usr/share/uhd/images instead of what this says:
" The images will be downloaded to: /opt/local/share/uhd/images " ?

**Comment by DeerStalker25** [comment]
I appreciate your quick response. Its also possible that I broke something. Thank you for looking into it! I was having a bunch of errors. I'm using DragonOS so I'm wondering if my install of my distro is broken or perhaps its a bit different from Ubuntu. I'm trying to make it happen on an RPi 4b 8Gb with Ubuntu 22.04 LTS.

**Comment by lmesserStep** [comment]
> I appreciate your quick response. Its also possible that I broke something. Thank you for looking into it! I was having a bunch of errors. I'm using DragonOS so I'm wondering if my install of my distro is broken or perhaps its a bit different from Ubuntu. I'm trying to make it happen on an RPi 4b 8Gb with Ubuntu 22.04 LTS.

Absolutely! 

Try replacing the bin with my binary and see if it works in that directory. I think I may need to update the readme. 

Are you able to prove the device with the util?

**Comment by DeerStalker25** [comment]
No, it gives me the error "Error: RuntimeError: fx3 is in state 5". I've tried replacing the .bin with yours both graphically and CLI. Same error as before. When I clone your repo I get "Cloning into 'LibreSDRB210'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists". 

So I git cloned the code link then like I said tried GUI and CLI to replace the .bin.    When I probe it points to /usr/share/uhd/images/usrp_b210_fpga.bin , so it SHOULD work...

**Comment by lmesserStep** [comment]
> No, it gives me the error "Error: RuntimeError: fx3 is in state 5". I've tried replacing the .bin with yours both graphically and CLI. Same error as before. When I clone your repo I get "Cloning into 'LibreSDRB210'...
> 
> git@github.com: Permission denied (publickey).
> 
> fatal: Could not read from remote repository.
> 
> 
> 
> Please make sure you have the correct access rights
> 
> and the repository exists". 
> 
> 
> 
> So I git cloned the code link then like I said tried GUI and CLI to replace the .bin.    When I probe it points to /usr/share/uhd/images/usrp_b210_fpga.bin , so it SHOULD work...

Just curious, which clone did you buy?

**Comment by DeerStalker25** [comment]
Very similar to yours, except I have the 7A200T FPGA. Here's the link: 
https://www.ebay.com/itm/126592924523

**Comment by DeerStalker25** [comment]
So I just tried compiling on my RPi4B w Ubuntu, it still gives me permission denied when I try to clone your repo.

**Comment by DeerStalker25** [comment]
I'm guessing its just a matter of I have a different FPGA than you do. Like I said, I have the AT200T. If I'm not mistaken, you have the AT100T.

**Comment by lmesserStep** [comment]
Those will have different logics. Would you be able to confirm with your seller if they will be able to share their binaries or Vivado projects? They may have a solution already.

**Comment by DeerStalker25** [comment]
They sent me a Vivado file but I couldn't find any .bin files in the files.

**Comment by DeerStalker25** [comment]
Scratch that, I downloaded it on Windows, extracted the RAR file and found the .bin. I renamed it to match the usrp_b210_fpga.bin and replaced that in /usr/share/udh/image, probed then had same error.

**Comment by DeerStalker25** [comment]
Wait. I got it to work w the Vivado .bin! Like I said I attempted to say, I had to use WinRAR to extract the file from the seller then I found the .bin, renamed it to match the usrp_b210_fpga.bin, sudo cp like in your instructions, the probed and it looks identical to your print out. Maybe Ill learn to create a GH page with my specific device. I appreciate your time and help. Thanks again! I look forward to more videos!
